app/models/cloudcredential.js

```diff
@@ -74,6 +75,37 @@ const cloudCredential = Resource.extend({
       });
     }
   },
+  validationErrors() {
+    let errors = [];
+
+    if (this.isOCI) {
+      if (!this.get('ocicredentialConfig.tenancyId')) {
+        errors.push(requiredError('modalAddCloudKey.oci.tenancyOcid.label'));
+      }
+
+      if (!this.get('ocicredentialConfig.region')) {
+        errors.push(requiredError('modalAddCloudKey.oci.authRegion.label'));
+      }
+
+      if (!this.get('ocicredentialConfig.userId')) {
+        errors.push(requiredError('modalAddCloudKey.oci.userOcid.label'));
+      }
+
+      if (!this.get('ocicredentialConfig.fingerprint')) {
+        errors.push(requiredError(`modalAddCloudKey.oci.userFingerprint.label`));
+      }
+
+      if (!this.get('ocicredentialConfig.privateKeyContents')) {
+        errors.push(requiredError('modalAddCloudKey.oci.secretKey.label'));
+      }
+
+      return errors;
```

Comment:
OK - so this code explicitly tests for the individual fields and ignores passphrase.  make sense.